ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -106,6 +107,7 @@
 
   private final PeerDiscoveryAgent agent;
   private final PeerBlacklist peerBlacklist;
+  private final NodeWhitelistController nodeWhitelistController;
```

Comment:
Suggestion: s/nodeWhiteListController/nodeWhitelist
Reason being, the term `Controller` is unnecessary, brings no additional contextual meaning.